### ISSUE TITLE
docker not lauching

### ISSUE THREAD

**Issue by YoShiiro** [030/n3dr#74]
Hello guy,

I've try to use your soft, because i want to download some repository from our older nexus3 (test) to our new production nexus3

but when i launch your docker through docker-compose and the docker creating but i got an error and no docker stay alive. I don't know if i'm doing something wrong or not ? i will link you my config / steps

1/
create a /opt/n3dr/.n3dr.yaml (with chmod 400)
create a /opt/n3dr/download/ (folder)

2/ create my docker-compose.yml
```
version: '3.7'
services:
  n3dr:
    image: utrecht/n3dr:3.0.0
    volumes:
      - /opt/n3dr/.n3dr.yaml:/home/n3dr/.n3dr.yaml
      - /opt/n3dr/download:/download
    command: repositories -n http://myservernexusonlan:8081/nexus/ -u admin -v ${NEXUS_API_VERSION} -b
```
and when i do my docker-compose up i got this ?
```
docker-compose up
WARNING: The NEXUS_API_VERSION variable is not set. Defaulting to a blank string.
Recreating n3dr_n3dr_1 ... done
Attaching to n3dr_n3dr_1
n3dr_1  | Error: unknown shorthand flag: 'v' in -v
n3dr_1  | Usage:
n3dr_1  |   n3dr repositories [flags]
n3dr_1  | 
n3dr_1  | Flags:
n3dr_1  |   -b, --backup   Backup artifacts from all repositories
n3dr_1  |   -c, --count    Count the number of repositories
n3dr_1  |   -h, --help     help for repositories
n3dr_1  |   -a, --names    Print all repository names
n3dr_1  | 
n3dr_1  | Global Flags:
n3dr_1  |   -d, --debug             Enable debug logging
n3dr_1  |   -p, --n3drPass string   The Nexus3 password
n3dr_1  |   -n, --n3drURL string    The Nexus3 URL
n3dr_1  |   -u, --n3drUser string   The Nexus3 user
n3dr_1  | 
n3dr_1  | unknown shorthand flag: 'v' in -v
n3dr_n3dr_1 exited with code 1
```

and i got no more logs nothing.

**Comment by YoShiiro** [comment]
So i advance, i create the docker using the docker file ;). But when i try to connect to my nexus nothing appen even if i put right credential 

i add context path "/nexus" (that we keep from old nexus) because we are migrating from 2 to 3

```
docker run -v /opt/n3dr/.n3dr.yaml:/home/n3dr/.n3dr.yaml -v /opt/n3dr/download:/download n3dr repositories -n http://mynexus:8081/nexus/ -c -u admin -v v1 -d
```
Answer : 
```time="2019-07-16T15:50:36Z" level=debug msg="Consult repositories URL: http://myservernexus:8081/nexus/" func=n3dr/cli.Nexus3.repositories file="/go/n3dr/cli/repositories.go:15"
0```

**Comment by 030** [comment]
Could you use the latest n3dr? I.e. 3.1.0.

**Comment by 030** [comment]
@YoShiiro Did 3.1.0 solve the issue?

**Comment by 030** [comment]
@YoShiiro Could you use the command line tool and check whether the issue persists?

**Comment by 030** [comment]
Suggestion is to remove docker support as it contains too many issues.

**Comment by 030** [comment]
Docker support removed

**Comment by 030** [comment]
https://github.com/wagoodman/dive

**Comment by 030** [comment]
Resolved in 6.0.3.
